dts/bindings/base/base.yaml

```diff
@@ -97,3 +97,13 @@ properties:
       required: false
       type: string-array
       description: Provided names of IO channel specifiers
+
+    mboxes:
+      required: false
+      type: phandle-array
+      description: mailbox channels specifiers
```

Comment:
To be more aligned to the Zephyr naming can we say `mailbox / IPM channel specifiers`?